README.md

```diff
@@ -105,8 +105,8 @@ export GCLOUD_PROFILER_CONFIG=./path/to/your/profiler/configuration.js
 
 The profiler writes log statements to the console log for diagnostic purposes.
 By default, the log level is set to warn. You can adjust this by setting
-`logLevel` in the config. Setting `logLevel` to 0 will disable logging, 1 sets
-log level to error, 2 sets it to warn, 3 sets it to info, and 4 sets it to
+`logLevel` in the config. Setting `logLevel` to 0 will disable logging (default),
+1 sets log level to error, 2 sets it to warn, 3 sets it to info, and 4 sets it to
```

Comment:
Confirmed by checking configs. 

2, not 0, is default.

Reply:
Noting that this needs to be addressed.

Reply:
Noting this should be addressed.

Reply:
done